src/nviz/view.py

```diff
@@ -53,7 +53,7 @@ def view_zarr_with_napari(
         for label_name in sorted(frame_zarr["labels"].keys(), reverse=True):
             viewer.add_labels(
                 frame_zarr["labels"][label_name]["0"][:],
-                name=f"{label_name} (label)",
+                name=f"{label_name}",
```

Comment:
Big fan of this change! It's often confusing when software handles naming differently than what's expected :)

Reply:
Thanks, this was removed as it became a duplicate of text which was added. Generally I opted to add a small bit of text to each label so it appears as a label in Napari properly (to differentiate from image layers).